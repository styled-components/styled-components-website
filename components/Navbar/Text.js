import styled, { css } from 'styled-components'

import rem from '../../utils/rem'
import { mobile } from '../../utils/media'

const Text = styled.div`
  display: none;
  margin-left: ${rem(15)};
  margin-top: ${rem(5)};
  font-size: ${rem(18)};

  ${mobile(css`
    display: block;
  `)}
`

export default Text
