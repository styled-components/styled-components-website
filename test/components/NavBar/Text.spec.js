import React from 'react';
import renderer from 'react-test-renderer';
import Text from '../../../components/Nav/Text';

test('Text renders correctly', () => {
  const tree = renderer.create(<Text />).toJSON();

  expect(tree).toMatchSnapshot();
});
