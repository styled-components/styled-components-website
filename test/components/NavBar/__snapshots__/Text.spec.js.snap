// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders correctly 1`] = `
.c0 {
  display: none;
  margin-left: 0.8333333333333334rem;
  margin-top: 0.2777777777777778rem;
  font-size: 1rem;
}

@media (max-width:62.5em) {
  .c0 {
    display: block;
  }
}

<div
  className="c0"
/>
`;
